app/black_spots/models.py

```diff
@@ -47,3 +49,16 @@ class RoadSegmentsShapefile(AshlarModel):
 
     #: Path to gzipped shapefile
     shp_tgz = models.FileField(upload_to='road_segments/')
+
+
+class BlackSpotTrainingCsv(AshlarModel):
+    """Model to track blackspot training csvs"""
+
+    #: Path to csvs
+    csv = models.FileField(upload_to='training/blackspot/')
+
+
+class ForecastTrainingCsv(AshlarModel):
```

Comment:
It might be good to make this `LoadForecastTrainingCsv`; I was a bit confused about the meaning since the blackspot script itself outputs a `forecasts.csv` by default.